docs/Installing.asciidoc

```diff
@@ -84,6 +84,33 @@ zypper in openQA-bootstrap
 systemd-run -tM openqa1 /bin/bash # start a shell in the container
 -------------------------------------------------------------------------------
 
+== Container based setup
+[id="container_setup"]
+
+Containers are provided for both the webUI and worker.
```

Comment:
```suggestion
Containers are provided for both the web UI and worker.
```

We mainly use that spelling so for the sake of consistency. Besides, if you would write it out ("user interface") you would add a space as well so in my opinion it only makes sense to add a space.

Reply:
you are right. Thanks for your attention to detail.